docs/implicit_diff.rst

```diff
@@ -16,25 +18,45 @@ is usually done either by implicit differentiation or by autodiff through an
 algorithm's unrolled iterates.
 
 
+JAXopt solvers
+--------------
 
-.. topic:: Examples
+All solvers in JAXopt support implicit differentiation **out-of-the-box**.
+Most solvers have an ``implicit_diff=True|False`` option. When set to ``False``,
+unrolling through the iterates is used instead of implicit differentiation.
```

Comment:
before this approach was referred to as "autodiff of unrolled iterations". I would make it consistent to avoid confusion.